tests/Microsoft.Identity.Test.Integration.netfx/Infrastructure/TargetFramework.cs

```diff
@@ -6,46 +6,82 @@
 
 namespace Microsoft.Identity.Test.Integration.Infrastructure
 {
+    public class RunOnAttribute : TestMethodAttribute
+    {
+        private TargetFwk _tfm;
+
+        public RunOnAttribute(TargetFwk tfm)
```

Comment:
nit: rename tfm to tfms (or frameworks)? since it's a flag and can contain multiple values?